src/main/java/erb/CurrentProject.java

```diff
@@ -1,27 +1,27 @@
 /**
  * CurrentProject.java
- * Created on 13.02.2003, 13:16:52 Alex
- * Package: net.sf.memoranda
+ * Created on 13.02.2003, 13:16:52 
+ * Package: net.sf.erb
  *
- * @author Alex V. Alishevskikh, alex@openmechanics.net
- * Copyright (c) 2003 Memoranda Team. http://memoranda.sf.net
+ * @author Mason Cole, Steven Foust, Devin Forrester, Veronica Kanczes, Jacob Wallert
+ * Copyright (c) 2003 Einstein Rosen Bus Team. 
```

Comment:
may want to change date to 2018 but no biggie
```suggestion
 * Copyright (c) 2018 Einstein Rosen Bus Team. 
```